File: CS/CarList.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace summary
{
    public static class CarList
    {
        public static BindingList<Car> GetList()
        {
            var list = new BindingList<Car>();
            list.Add(new Car() { ID = 1, Trademark = "Mercedes-Benz", Model = "SL500 Roadster", HP = 302, Liter = 4.966 });
            list.Add(new Car() { ID = 2, Trademark = "Mercedes-Benz", Model = "CLK55 AMG Cabriolet", HP = 342, Liter = 5.439 });
            list.Add(new Car() { ID = 3, Trademark = "Mercedes-Benz", Model = "C230 Kompressor Sport Coupe", HP = 189, Liter = 1.796 });
            list.Add(new Car() { ID = 4, Trademark = "BMW", Model = "530i", HP = 225, Liter = 3 });
            list.Add(new Car() { ID = 5, Trademark = "Rolls-Royce", Model = "Corniche", HP = 325, Liter = 6.75 });
            list.Add(new Car() { ID = 6, Trademark = "Jaguar", Model = "S-Type 3.0", HP = 235, Liter = 3 });
            list.Add(new Car() { ID = 7, Trademark = "Cadillac", Model = "Seville", HP = 275, Liter = 4.6 });
            list.Add(new Car() { ID = 8, Trademark = "Cadillac", Model = "DeVille", HP = 275, Liter = 4.6 });
            list.Add(new Car() { ID = 9, Trademark = "Lexus", Model = "LS430", HP = 290, Liter = 4.3 });
            list.Add(new Car() { ID = 10, Trademark = "Lexus", Model = "GS 430", HP = 300, Liter = 4.3 });
            list.Add(new Car() { ID = 11, Trademark = "Ford", Model = "Ranger FX-4", HP = 135, Liter = 2.3 });
            list.Add(new Car() { ID = 12, Trademark = "Dodge", Model = "Ram 1500", HP = 215, Liter = 3.7 });
            list.Add(new Car() { ID = 13, Trademark = "GMC", Model = "Siera Quadrasteer", HP = 200, Liter = 4.3 });
            list.Add(new Car() { ID = 14, Trademark = "Nissan", Model = "Crew Cab SE", HP = 143, Liter = 2.4 });
            list.Add(new Car() { ID = 15, Trademark = "Toyota", Model = "Tacoma S-Runner", HP = 190, Liter = 3.4 });
            return list;
        }
    }
    public class Car
    {
        public int ID { get; set; }
        public string Trademark { get; set; }
        public string Model { get; set; }
        public Int16 HP { get; set; }
        public double Liter { get; set; }
    }
}

File: VB/CarList.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace summary
    Public NotInheritable Class CarList

        Private Sub New()
        End Sub

        Public Shared Function GetList() As BindingList(Of Car)
            Dim list = New BindingList(Of Car)()
            list.Add(New Car() With { _
                .ID = 1, _
                .Trademark = "Mercedes-Benz", _
                .Model = "SL500 Roadster", _
                .HP = 302, _
                .Liter = 4.966 _
            })
            list.Add(New Car() With { _
                .ID = 2, _
                .Trademark = "Mercedes-Benz", _
                .Model = "CLK55 AMG Cabriolet", _
                .HP = 342, _
                .Liter = 5.439 _
            })
            list.Add(New Car() With { _
                .ID = 3, _
                .Trademark = "Mercedes-Benz", _
                .Model = "C230 Kompressor Sport Coupe", _
                .HP = 189, _
                .Liter = 1.796 _
            })
            list.Add(New Car() With { _
                .ID = 4, _
                .Trademark = "BMW", _
                .Model = "530i", _
                .HP = 225, _
                .Liter = 3 _
            })
            list.Add(New Car() With { _
                .ID = 5, _
                .Trademark = "Rolls-Royce", _
                .Model = "Corniche", _
                .HP = 325, _
                .Liter = 6.75 _
            })
            list.Add(New Car() With { _
                .ID = 6, _
                .Trademark = "Jaguar", _
                .Model = "S-Type 3.0", _
                .HP = 235, _
                .Liter = 3 _
            })
            list.Add(New Car() With { _
                .ID = 7, _
                .Trademark = "Cadillac", _
                .Model = "Seville", _
                .HP = 275, _
                .Liter = 4.6 _
            })
            list.Add(New Car() With { _
                .ID = 8, _
                .Trademark = "Cadillac", _
                .Model = "DeVille", _
                .HP = 275, _
                .Liter = 4.6 _
            })
            list.Add(New Car() With { _
                .ID = 9, _
                .Trademark = "Lexus", _
                .Model = "LS430", _
                .HP = 290, _
                .Liter = 4.3 _
            })
            list.Add(New Car() With { _
                .ID = 10, _
                .Trademark = "Lexus", _
                .Model = "GS 430", _
                .HP = 300, _
                .Liter = 4.3 _
            })
            list.Add(New Car() With { _
                .ID = 11, _
                .Trademark = "Ford", _
                .Model = "Ranger FX-4", _
                .HP = 135, _
                .Liter = 2.3 _
            })
            list.Add(New Car() With { _
                .ID = 12, _
                .Trademark = "Dodge", _
                .Model = "Ram 1500", _
                .HP = 215, _
                .Liter = 3.7 _
            })
            list.Add(New Car() With { _
                .ID = 13, _
                .Trademark = "GMC", _
                .Model = "Siera Quadrasteer", _
                .HP = 200, _
                .Liter = 4.3 _
            })
            list.Add(New Car() With { _
                .ID = 14, _
                .Trademark = "Nissan", _
                .Model = "Crew Cab SE", _
                .HP = 143, _
                .Liter = 2.4 _
            })
            list.Add(New Car() With { _
                .ID = 15, _
                .Trademark = "Toyota", _
                .Model = "Tacoma S-Runner", _
                .HP = 190, _
                .Liter = 3.4 _
            })
            Return list
        End Function
    End Class
    Public Class Car
        Public Property ID() As Integer
        Public Property Trademark() As String
        Public Property Model() As String
        Public Property HP() As Int16
        Public Property Liter() As Double
    End Class
End Namespace
